app/reducers/quotes.js

```diff
@@ -80,12 +83,14 @@ export const selectQuoteById = createSelector(
 export const selectSortedQuotes = createSelector(
   selectQuotes,
   (state, props) => ({ filter: props.filter }),
-  (quotes, query) => {
+  (_, props) => props && props.pagination,
+  (quotes, query, pagination) => {
     return quotes
       .filter(
         (quote) =>
           typeof quote !== 'undefined' &&
-          quote.approved === (query.filter !== 'unapproved')
+          quote.approved.toString() ===
+            (pagination.query.approved && pagination.query.approved.toString())
```

Comment:
Do you think it would be cleaner to add an optinal chain to this?😄 `pagination.query.approved?.toString()`

Reply:
And ref my other comment, default `true`?

Reply:
Hmm. Tbh, after thinking, we don't need this filtering, since it is already filtered on the pagination :eyes:  Will remove it

Reply:
Hmmmm. The `approve` and `unapprove` logic will then be kinda strange.... :/